### ISSUE TITLE
bug: [Dev Vers.] `Theme` is incompatible with version 18.38.44

### ISSUE THREAD

**Issue by cyberboh** [ReVanced/revanced-patches#3157]
### Bug description

The issue is warning msg about Theme is incompatible with version 18.38.44.

1. Download [YT 18.38.44](https://www.apkmirror.com/apk/google-inc/youtube/youtube-18-38-44-release/youtube-18-38-44-2-android-apk-download/)
2. Patch with CLI
```
java -jar revanced-cli.jar patch -p -o Install\YouTube_ReVanced.apk -b revanced-patches.jar --keystore=ks.keystore -m revanced-integrations.apk YouTube.apk
```
3. See the 'Theme' warning msg

### Error logs

```shell
java -jar revanced-cli.jar patch -p -o Install\YouTube_ReVanced.apk -b revanced-patches.jar --keystore=ks.keystore -m revanced-integrations.apk YouTube.apk
INFO: Loading patches
INFO: Decoding app manifest
WARNING: Theme is incompatible with version 18.38.44. This patch is only compatible with version 18.37.36, 18.38.44
INFO: Custom branding excluded
INFO: Remove player controls background excluded
INFO: Export all activities excluded
INFO: Remove screen capture restriction excluded
INFO: Remove screenshot restriction excluded
INFO: Spoof Wi-Fi connection excluded
INFO: Predictive back gesture excluded
INFO: Override certificate pinning excluded
INFO: Change package name excluded
INFO: Enable Android debugging excluded
INFO: Setting patch options
<SNIP>
```

### Solution

Maybe caused by this:
https://github.com/ReVanced/revanced-patches/commit/90d5877950095b7abacdca979bc7ad709192eee2#r130104706

### Additional context

CLI [v4.0.2](https://github.com/ReVanced/revanced-cli/releases/tag/v4.0.2)
Patches [v2.195.0-dev.2](https://github.com/ReVanced/revanced-patches/releases/tag/v2.195.0-dev.2)
Inte [v0.120.0-dev.4](https://github.com/ReVanced/revanced-integrations/releases/tag/v0.120.0-dev.4)

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by Ushie** [comment]
This has been fixed already by ae7c60e2bdbe7671af811b5d2a7fae0d813dc909

**Comment by oSumAtrIX** [comment]
This is already fixed
